libass/ass.h

```diff
@@ -24,7 +24,7 @@
 #include <stdarg.h>
 #include "ass_types.h"
 
-#define LIBASS_VERSION 0x01502000
+#define LIBASS_VERSION 0x01503000
```

Comment:
This is a release version number. For API changes between releases, just add 1.

Reply:
Done.

Reply:
In the discussion regarding hotfix releases, a suggestion was to start the increase for new features at the second nibble. Here `0x01600010`, so that a potential future hotfix release with just a critical fix backported can use `0x01600001` and then be released as 0.16.0.1, I guess. *(Side note: Maybe it would also be a good idea to move out of `0.x.x` so the third number can be used for hotfix releases instead of needing 4 numbers?)*